airdrop/processor/default_airdrop.py

```diff
@@ -16,6 +16,7 @@ def __init__(self, airdrop_id, airdrop_window_id=None):
         self.claim_signature_data_format = ["string", "uint256", "uint256", "address", "uint256", "uint256", "address",
```

Comment:
Let's revisit this approach of a separate airdrop class and a separate rewards class. For now approving as we have an issue